lib/matplotlib/axes/_axes.py

```diff
@@ -2705,11 +2705,11 @@ def errorbar(self, x, y, yerr=None, xerr=None,
 
         Parameters
         ----------
-        x : scalar
-        y : scalar
+        x : scalar or array-like
```

Comment:
I have seen it used that way pretty consistently in `numpy`, where a scalar gets converted to an array of shape `(0,)` when necessary.